templates/course-theme/lesson-quiz-notice.php

```diff
@@ -20,7 +20,7 @@
 	 */
 	function sensei_lesson_quiz_notices_map( $notice ) {
 		?>
-		<div class="sensei-lms-notice sensei-course-theme-lesson-quiz-notice">
+		<div class="sensei-course-theme__frame sensei-lms-notice sensei-course-theme-lesson-quiz-notice">
```

Comment:
Including the notice in `__frame`, since we'll want this to ignore content styles.

Reply:
Yes, because we want to keep this separate from the content styles (not pick up fonts, etc from there). We might call the `__frame` context differently at some point, but going forward I think we'll want to separate a few blocks like this from the content — it was constant trouble in the past to keep Sensei elements compatible and looking good with every theme.

Reply:
> it was constant trouble in the past to keep Sensei elements compatible and looking good with every theme

I think it wouldn't be a problem in this case where we had to support only one theme. I'm still mainly concerned with the `* { all: unset; display: revert; }`. I think it could cause us many issues.

Reply:
We can scope the content styles of our theme to just a content wrapper element, so it doesn't affect the frame, but it would be nice to keep the possibility of these blocks ever working with FSE and users being able to create custom templates with them. And that means keeping in mind to support any theme. Of course the question then is, should, for example, the notices block, use controlled Sensei styles, or pick up the theme styles.

Right now adding this class just makes sure that it's using the interface styles, eg. Inter font, without having to add those styles all over again for this block too.

Reply:
About the `*`, I was checking [this reference](https://elad2412.github.io/the-new-css-reset/). There's a different selector to avoid some issues (our svgs work after updating the selector to this): `*:where(:not(iframe, canvas, img, svg, video):not(svg *, symbol *))`.

There's only a new issue with that in the logo because it's skipping images too. The issue makes sense because it's setting width/height in the image through the `wp_get_attachment_image`. We could just add an extra property in that specific style to set `width: auto`.

Reply:
Nice, I'll try that in the content styles PR.